test/unit_tests/braket/analog_hamiltonian_simulator/test_rydberg_simulator.py

```diff
@@ -0,0 +1,182 @@
+import numpy as np
+import pytest
+from braket.ir.ahs.program_v1 import Program
+from braket.task_result.analog_hamiltonian_simulation_task_result_v1 import (
+    AnalogHamiltonianSimulationTaskResult,
+)
+
+from braket.analog_hamiltonian_simulator.rydberg.constants import RYDBERG_INTERACTION_COEF
+from braket.analog_hamiltonian_simulator.rydberg.rydberg_simulator import RydbergAtomSimulator
+from braket.analog_hamiltonian_simulator.rydberg.rydberg_simulator_unit_converter import (
+    convert_unit,
+)
+
+device = RydbergAtomSimulator()
+
+
+pi = np.pi
+
+a = 3
+Omega = 12 * pi
```

Comment:
nit: the convention is that variables are in lower case, but free feel to keep if there is a reason

Reply:
Yeah, since if we write the formulas in Latex, Omega and omega means different things, and we exclusively use Omega for Rabi frequency, let us keep the capital D here.

Reply:
is there any ambiguity that it might be referring to lowercase omega? If not, then I suggest we switch to lowercase for this and Deltas. There are many cases where uppercase words get lowercased to become variable names. We shouldn't be relying on case to disambiguate

Reply:
Also, instead of Greek letters, how about what the quantity actually _is_?

Reply:
There will be ambiguity if we use lowercase omega instead of uppercase Omega, because in our document, notebook etc, we use Omega to denote the physical quantity _Rabi frequency_, which seems too long to be a variable name. How about we put the formula at the top of this (and other unit test) file to define what this Greek letters mean. Would that help?

Reply:
`rabi_frequency` is a perfectly acceptable variable name. I would even prefer `rabi_freq` over `Omega`

Reply:
Gotya. Let me change the names.